cmd/tracee-ebpf/main.go

```diff
@@ -135,9 +135,18 @@ func main() {
 			}
 			cfg.Filter = &filter
 
-			containerMode := (cfg.Filter.ContFilter.Enabled() && cfg.Filter.ContFilter.Value()) ||
+			containerMode := printer.ContainerModeDisabled
+
+			if (cfg.Filter.ContFilter.Enabled() && cfg.Filter.ContFilter.Value()) ||
```

Comment:
maybe we could extract a method to make this read a little better? 

```
if checkAnyContainerFlagEnabled(cfg)
```

Reply:
Had that on my eye, WDYT about declaring it as a variable rather than a method?
Like so:
```
isAnyContainerFlagEnabled := ...
```

Reply:
I did something to that code in multiscope:
 
- https://github.com/aquasecurity/tracee/pull/2238/files#diff-fc55c933c348f5f4fd12cc39c7c7ad6e0c8625dcfd03520d54b0b1aae8c12cd6R147-R149
- https://github.com/aquasecurity/tracee/pull/2238/files#diff-70859fceede6c73f664e1a1f54783ef93c29fccefbf64230aaaf76071843cf55R217-R225
- https://github.com/aquasecurity/tracee/pull/2238/files#diff-70859fceede6c73f664e1a1f54783ef93c29fccefbf64230aaaf76071843cf55R361-R364

@josedonizetti @NDStrahilevitz if you are touching filters, printer, parse etc, let's keep an eye on the multiscope. There's a bunch of changes there to those areas.

Reply:
@geyslan I'll keep you pinged on any new filter PRs I add, but I don't think we should wait on minor changes for them to be introduced in a later much larger PR.

Reply:
> Had that on my eye, WDYT about declaring it as a variable rather than a method? Like so:
> 
> ```
> isAnyContainerFlagEnabled := ...
> ```

@NDStrahilevitz definitely helps, but i think a method helps with LOC for the main too. But either way work.